src/module/form.js

```diff
@@ -0,0 +1,185 @@
+    checker(tasks, box, checked);
+    // remove all marked tasks
+
+    const clear = document.querySelector('.okay');
+    clear.addEventListener('click', () => {
+      const each = document.querySelectorAll('#completed');
+      each.forEach((e) => {
+        if (e.id === 'completed') {
+          e.remove();
+        }
+        tasks = tasks.filter((t) => (t.completed !== true));
+        updateLocalStorage(tasks);
+      });
+    });
+  });
+  // for reload
+
+  window.addEventListener('load', () => {
+    const reshow = JSON.parse(localStorage.getItem('store')) || [];
+    tasks.push(...tasks, ...reshow);
+    tasks.map((task) => (display(task, list)));
+    updateLocalStorage(tasks);
+    // edit task
+    const pen = document.querySelectorAll('#more');
+    pen.forEach((p, y) => {
+      p.addEventListener('click', () => {
+      // change the icon
+
+        p.parentElement.classList.add('ic');
+        const grand = p.parentElement.parentElement;
+        // remove task
+
+        const times = document.querySelectorAll('#trash');
+        times.forEach((time, n) => {
+          time.addEventListener('click', () => {
+            time.parentElement.parentElement.remove();
+            tasks.forEach((t, m) => {
+              if (n === m) {
+                tasks = tasks.filter((tee) => (tee.index !== t.index));
+                // changeindexs
+                tasks.forEach((tep, num) => {
+                  tep.index = num + 1;
+                });
+                updateLocalStorage(tasks);
+              }
+            });
+          });
+        });
+        // set the input to readonly
+
+        const edit = grand.querySelector('#tasks');
+        edit.removeAttribute('readonly');
+        edit.focus();
+        // change the description in array and LS
+
+        edit.addEventListener('keyup', () => {
+          tasks.forEach((e, z) => {
+            if (y === z) {
+              e.description = edit.value;
+              updateLocalStorage(tasks);
+            }
+          });
+        });
+        // off the edit function
+        const worr = grand.querySelector('#worr');
+        worr.addEventListener('submit', (e) => {
+          e.preventDefault();
+          p.parentElement.classList.remove('change');
+          edit.setAttribute('readonly', 'readonly');
+        });
+      });
+      // check on button
+
+      const box = document.querySelectorAll('.goes');
+      const checked = document.querySelectorAll('#checked');
+      checker(tasks, box, checked);
+      // remove all marked tasks
+      const clear = document.querySelector('.okay');
+      clear.addEventListener('click', () => {
+        const each = document.querySelectorAll('#completed');
+        each.forEach((e) => {
+          if (e.id === 'completed') {
+            e.remove();
+          }
+          tasks = tasks.filter((t) => (t.completed !== true));
+          updateLocalStorage(tasks);
+        });
```

Comment:
- [OPTIONAL] Kindly make sure once you delete the complete tasks in the local storage, you update the index of the rest task. At this moment, when completed tasks are deleted, the index of the rest are being updated. Kindly fix that.

   _**LOCAL STORAGE  BEFORE DELETION**_
     ![image](https://user-images.githubusercontent.com/87197412/215089282-01878286-097b-4130-b010-660e6ef036d7.png)

   _**LOCAL STORAGE  AFTER DELETION**_
     ![image](https://user-images.githubusercontent.com/87197412/215089366-fb292b07-fc8f-4fd0-83c3-a5be7ae35ac1.png)